end-to-end-tests/.eslintrc.json

```diff
@@ -0,0 +1,89 @@
+{
+  "env": {
+    "browser": true,
+    "es2021": true
+  },
+  "parser": "@typescript-eslint/parser",
+  "extends": [
+    "plugin:react/recommended",
+    "plugin:json/recommended",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:import/typescript",
+    "prettier"
+  ],
+  "parserOptions": {
+    "ecmaFeatures": {
+      "jsx": true
+    },
+    "ecmaVersion": 12,
+    "sourceType": "module",
+    "requireConfigFile": false
+  },
+  "plugins": [
+    "react",
+    "prettier",
+    "json",
+    "import"
+  ],
+  "settings": {
+    "import/resolver": {
+      "node": {
+        "extensions": [
+          ".js",
+          ".jsx",
+          ".ts",
+          ".tsx"
+        ]
+      }
+    }
+  },
+  "rules": {
+    "react/jsx-filename-extension": [
+      1,
+      {
+        "extensions": [
+          ".js",
+          ".jsx",
+          ".tsx"
+        ]
+      }
+    ],
+    "react/prop-types": [
+      0
+    ],
+    "react/destructuring-assignment": [
+      0
+    ],
+    "react/jsx-props-no-spreading": "off",
+    "prettier/prettier": "error",
+    "import/no-anonymous-default-export": "off",
+    "import/no-extraneous-dependencies": [
+      "error",
+      {"devDependencies": true}
+    ],
```

Comment:
There were a handful of  `no-extraneous-dependencies` issues here are I just went ahead and disabled them all. Although, we probably will want this ignore to be a little more specific.

The main issues are:
`oscal-editor-deployment/end-to-end-tests/cypress.config.ts
   2:26  error    'cypress' should be listed in the project's dependencies, not devDependencies`

`oscal-editor-deployment/end-to-end-tests/cypress/support/commands.js
  26:1  error  '@testing-library/cypress' should be listed in the project's dependencies, not devDependencies`